docs/user-api/backend-api/resources/tracking/tracker/index.md

```diff
@@ -1214,89 +1214,156 @@ Gets all integrated tracker models (from "models" table).
 {
-    "id": 166,
-    "code": "tt1_wp",
-    "type": "vehicle",
-    "name": "WondeProud TT1",
-    "id_type": "10,2",
-    "has_phone": true,
-    "has_apn_settings": true,
-    "register": true,
-    "battery": {
-        "min_charge": 3.4,
-        "low_charge": 3.7,
-        "max_charge": 4.1
-    },
-    "altitude": true,
-    "satellites": true,
-    "gsm_level": true,
-    "gsm_network": true,
-    "gsm_roaming": true,
-    "has_detach_button": false,
-    "has_fuel_input": true,
-    "has_custom_fields": false,
-    "analog_inputs": 2,
-    "digital_inputs": 4,
-    "rs232_inputs": 0,
-    "digital_outputs": 4,
-    "track_control": "tt1",
-    "output_control": "default",
-    "special_control": "none",
-    "vendor": "WondeProud",
-    "rules": [
-        "offline",
-        "input_change",
-        "sos",
-        "sensor_range",
-        "speedup",
-        "route",
-        "track_change",
-        "inoutzone",
-        "battery_off"
-    ],
-    "inputs": ["analog_2"],
-    "state_fields": [],
-    "special_settings": ["none"],
-    "sms_control": [],
-    "has_led_control": false,
-    "has_location_request": true,
-    "has_gsm_lbs_location_request": true,
-    "has_chat": false,
-    "check_bundle": false,
-    "has_odometer": true
+  "id": 2450,
+  "vendor": "Navixy",
+  "code": "navixy_ngp",
+  "parent_code": null,
+  "type": "vehicle",
+  "name": "Navixy Generic Protocol",
+  "id_type": "ascii,6,64",
+  "has_phone": false,
+  "has_apn_settings": false,
+  "register": true,
+  "has_auto_registration": false,
+  "port": null,
+  "battery": {
+    "min_charge": 0.0,
+    "low_charge": 0.1,
+    "max_charge": 1.0
+  },
+  "altitude": true,
+  "satellites": true,
+  "gsm_level": true,
+  "gsm_network": true,
+  "gsm_roaming": true,
+  "has_detach_button": false,
+  "has_fuel_input": true,
+  "analog_inputs": 32,
+  "digital_inputs": 8,
+  "digital_outputs": 8,
+  "rs232_inputs": 0,
+  "track_control": "none",
+  "output_control": "default",
+  "special_control": "none",
+  "rules": [
+    "speedup",
+    "inoutzone",
+    "route",
+    "offline",
+    "track_change",
+    "sensor_range",
+    "driver_change",
+    "task_status_change",
+    "fuel_level_leap",
+    "distance_control",
+    "excessive_driving",
+    "excessive_parking",
+    "state_field_control",
+    "sos",
+    "input_change",
+    "output_change",
+    "idling_soft"
+  ],
+  "inputs": [
+    "analog_32",
+    "board_voltage",
+    "ext_temp_sensor_32",
+    "humidity_32",
+    "hw_mileage",
+    "lls_level_32",
+    "lls_temperature_32",
+    "temp_sensor"
+  ],
+  "state_fields": [
+    "event_id",
+    "hardware_key",
+    "moving",
+    "obd_vin"
+  ],
+  "special_settings": [
+    "none"
+  ],
+  "sms_control": [],
+  "connection": [{
+    "protocol": "NGP",
+    "transport": "HTTPS",
+    "url": "https://ngp-tracker.navixy.com"
+  }, {
+    "protocol": "NGP",
+    "transport": "MQTTS",
+    "url": "mqtts://ngp_device:secretword@mqtt.navixy.com:8883/ngp",
```

Comment:
Could you use non-real host in examples? Just for prevent spam from web crawlers.